src/python/WMComponent/JobAccountant/AccountantWorker.py

```diff
@@ -478,12 +478,12 @@ def handleJob(self, jobID, fwkJobReport):
             elif jobType == "LogCollect" and len(outputMap.keys()) == 0 and outputModules == set(['LogCollect']):
                 pass
             elif jobType == "Merge" and set(outputMap.keys()) == set(
-                    ['Merged', 'MergedError', 'logArchive']) and outputModules == set(['Merged', 'logArchive']):
+                    ['MergedRAW', 'MergedErrorRAW', 'logArchive']) and outputModules == set(['MergedRAW', 'logArchive']):
                 pass
             elif jobType == "Merge" and set(outputMap.keys()) == set(
-                    ['Merged', 'MergedError', 'logArchive']) and outputModules == set(['MergedError', 'logArchive']):
+                    ['MergedRAW', 'MergedErrorRAW', 'logArchive']) and outputModules == set(['MergedErrorRAW', 'logArchive']):
```

Comment:
I am not quite understand these conditions. Shouldn't original conditions also need to be kept (['Merged', 'MergedError'])? Is it only the Repack case? Maybe Alan should review. @amaltaro.
Also it seem above two conditions can be combined.

Reply:
Original case doesn't exist anymore since the output modules contain the RAW now at the end (per @amaltaro changes). Could be combined, but a combination wouldn't be any more readable code, at least not while being constrained to this exact use case.

The exact use case is RepackMerge, with one of the two possible output modules being dropped.